### ISSUE TITLE
Problem with ECMAScript modules import in nodejs and browser

### ISSUE THREAD

**Issue by NtsDK** [ramda/ramda#2988]
Hi!

I want run the same script both in nodejs and browser.
I use latest nodejs 13.9.0 which supports ECMAScript modules. And I use create-react-app to build application for browser.

`import * as R from 'ramda';` works well in browser but in nodejs it is imported as `{ default: { ...R functions}}`

`import R from 'ramda';` works well in nodejs but in browser it returns error 
`Attempted import error: 'ramda' does not contain a default export (imported as 'R').`

I'm not a fan of default imports so if it will work as `import * as R from 'ramda';` everywhere it will be awesome.

Is it possible to do something about it? Maybe some solution or workaround?

UPD. I made a very straightforward hack but it is ugly.
```
import * as R2 from 'ramda';

let R;

// need to hide the "default" call from webpack.
// Otherwise it returns: 'ramda' does not contain a default export 
function getter(obj, prop) {
  return obj[prop];
}

if (Object.keys(R2).length === 1) {
  R = getter(R2, 'default');
} else {
  R = R2;
}
```

**Comment by CrossEye** [comment]
@andarist: Do you have any suggestions here?

**Comment by Andarist** [comment]
I havent played much with them yet - but the answer is to provide conditional exports in package.json. Its a great opportunity for me to learn more about how to setup this correctly - so give me a couple of days and ill be back with a PR for this

**Comment by mikestopcontinues** [comment]
I just ran into this problem as well, trying to merge my api/app into a mono repo. I've been trying to find a babel workaround with little luck.  I'll post again if I hit on something.

**Comment by Andarist** [comment]
Huh, I've educated myself about this - but it seems that I forgot to report back my findings 😅 I'm going to play around with setting this up correctly for ramda in the following days.

**Comment by mikestopcontinues** [comment]
For the record, I wasn't able to find a babel-only solution, but switching from parcel to webpack allowed me to customize my build better. No more issues.

**Comment by CrossEye** [comment]
@mikestopcontinues: You might want to see Andarist's fixes in #2999.  That should be merged soon.

**Comment by mikestopcontinues** [comment]
Excellent! Thanks! It'll be nice to rollback my build hacks.
